scripts/teigaku_genzei/README.md

```
@@ -0,0 +1,239 @@
+## Command line interface
+
+`json_qa.py` is used to extract QA pairs from the above FAQ documents.
+```
+Usage: python json_qa.py <input_json_file> <output_csv_file>
+```
+where
+- input_json_file : A JSON file generated by `docparser_v2.py` (Docling) as a result of the extraction of text from a PDF.
+- output_csv_file : A CSV file which contains QA pairs. See ["Input file format"](#input-file-format).
+
+`json_glossary.py` are used to extract a glossary from the above FAQ documents.
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)